src/Exception/StatementDeserializationException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the xAPI package.
+ *
+ * (c) Christian Flothmann <christian.flothmann@xabbuh.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Xabbuh\XApi\Serializer\Exception;
+
+/**
+ * StatementDeserializationException.
```

Comment:
Please remove comments like this from the class docblocks. :) They don't add any real value.